src/apiCalls.tsx

```diff
@@ -0,0 +1,33 @@
+export const getWeddings = () => {
+  return fetch('http://localhost:8000/api/v1/weddings/weddings/')
+  .then(response => {
+    if(response.ok) {
+      return response.json()
+    } else {
+      throw Error
+    }
+  })
+  .catch(error => {
+    return "No weddings found"
+  })
+}
+
+export const postAWedding = (wedding: any) => {
+  return fetch('http://localhost:8000/api/v1/weddings/create/', {
+    method:'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(wedding)
+  })
+  .then(response => {
+    if(response.ok) {
+      return response
+    } else {
+      throw Error
```

Comment:
Should make some issues for user friendly error handling if the data can't be retrieved